tests/e2e/machine_pool_test.go

```diff
@@ -47,22 +44,23 @@ var _ = Describe("TF Test", func() {
 		})
 		Context("Author:amalykhi-High-OCP-64757 @OCP-64757 @amalykhi", func() {
 			It("Create a second machine pool", func() {
-				token := os.Getenv(con.TokenENVName)
+				By("Create a second machine pool")
 				replicas := 3
 				machine_type := "r5.xlarge"
-				name := "testmp14"
+				name := "testmp0"
```

Comment:
Let's make the name contains the case id, so if any resource leak we can know which case caused

Reply:
thanks